core-api/src/app.ts

```diff
@@ -11,7 +11,7 @@ import MealRouter from './routes/meal.route';
 
 class Core {
     public app: Express.Application;
-    public PORT = '8080';
+    public PORT = process.env.PORT;

```

Comment:
Change that for "process.env.PORT || 8080"

Reply:
done